.mpsrc

```diff
@@ -25,6 +25,7 @@
   "db_provider": "postgres",
   "connection_string": "postgresql://<USERNAME>:<PASSWORD>@localhost:5432/mpsdb?sslmode=no-verify",
   "instance_name": "localhost",
+  "auth_enabled": true,
```

Comment:
can we call this "web_auth_enabled" and move it next to "web_admin_user" so it is clear these are linked?

Reply:
done